share/completions/tmux.fish

```diff
@@ -17,6 +17,24 @@ function __fish_tmux_panes -d 'window panes'
     tmux list-panes -F '#W	window' 2>/dev/null
 end
 
+function __fish_tmux_buffers -d 'buffers'
+    tmux list-buffers -F '#{buffer_name}	#{buffer_sample}' 2>/dev/null
```

Comment:
Probably best done with `\t` rather than an embedded tab here

Reply:
Like this? I also included a commit replacing other embedded tabs.